### PR TITLE
Fixed bad airdrop unit bug

### LINKED ISSUE #309: Wrong units
We are using 1 avax = 1e9 when it should be 1e18.
https://github.com/ava-labs/avalanche-cli/blob/main/pkg/vm/evmSettings.go#L43

### PR DESCRIPTION
**Pull request by dasconnor**
Fixed wrong units bug in our airdrop logic. I also added a test to check this.

Closes #309

### CODE REVIEW

File: pkg/vm/airdrop.go [hunk 1]
```diff
@@ -13,6 +13,12 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
+const (
+	defaultAirdrop        = "Airdrop 1 million tokens to the default address (do not use in production)"
+	customAirdrop  string = "Customize your airdrop"
```
Comment (felipemadero) [inline pkg/vm/airdrop.go:18]: type seems not needed

File: pkg/vm/airdrop.go [hunk 2]
```diff
@@ -13,6 +13,12 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
+const (
```
Comment (felipemadero) [inline pkg/vm/airdrop.go:16]: not sure is this should be module level or function level consts

File: pkg/vm/airdrop.go (hunk 1 above)
Comment (dasconnor) [reply pkg/vm/airdrop.go]: My editor was complaining about a type checking thing, but I think it fixed itself so I removed this.

File: pkg/vm/airdrop.go (hunk 2 above)
Comment (dasconnor) [reply pkg/vm/airdrop.go:16]: I needed to do this to mock the prompt in the test.

**Review by felipemadero — APPROVED** [review]
LGTM

**Pull request merged**